### DISCUSSION TITLE
Electrolyte conductivity model equation

### DISCUSSION THREAD

**Discussion by lefile** [pybamm-team/PyBaMM#3434] [Q&A] [answered]
Hi!

Lithium current density in electrolyte in Full sub-model is calculated as (lines 55 - 57 in full_conductivity.py):
`i_e = (param.kappa_e(c_e, T) * tor) * (
            param.chiRT_over_Fc(c_e, T) * pybamm.grad(c_e) - pybamm.grad(phi_e)
        )`
while in paper "An Asymptotic Derivation of a Single Particle Model with Electrolyte" in Eq. 1b it is defined as:

![image](https://github.com/pybamm-team/PyBaMM/assets/35224046/02550e19-8983-4e6a-9ab7-c5f41fa147cb)

The difference is in natural logarithm of electrolyte concentration before it put in gradient. Am I missing something or is this a bug?

Best regards!

**Comment by valentinsulzer** [comment, marked as answer]
`grad(log(c)) = 1/c * grad(c)` by the chain rule, the `1/c` is included in `chiRT_over_FC`
